endpoint/webhook/tyntec-whatsapp/transformer.ts

```diff
@@ -111,34 +146,24 @@ interface ISessionStorageQuickReply {
 	quickReply: IDefaultQuickReply;
 }
 
-const createWhatsAppQuickReplies = (quickReplies: IDefaultQuickReply[], sessionStorage: any): string => {
-
-	// get previous quick replies from session storage 
-	let sessionquickReplyCurrentNumber: number = sessionStorage.quickReplyCurrentNumber || 0;
-	let sessionQuickReplies: ISessionStorageQuickReply[] = sessionStorage.quickReplies || [];
+const createWhatsAppQuickReplies = (quickReplies: IDefaultQuickReply[]): IWhatsAppQuickReply[] => {
 
-	// initialize empty text message bubble
-	let whatsAppQuickReplyMessage: string = "";
+	let whatsAppQuickReplies: IWhatsAppQuickReply[] = [];
 
 	for (let quickReply of quickReplies) {
-		// store the index to the session storage for further quick replies
-		sessionquickReplyCurrentNumber += 1;
-		sessionQuickReplies.push({
-			index: sessionquickReplyCurrentNumber,
-			quickReply
-		})
-		// add the quick reply to the text message bubble
-		// Example: 1. first quick reply
-		whatsAppQuickReplyMessage += `\n${sessionquickReplyCurrentNumber}. ${quickReply.title}`;
-
+		whatsAppQuickReplies.push({
+			type: "reply",
+			reply: {
+				payload: quickReply.payload,
+				title: quickReply.title
+			}
+		});
```

Comment:
This does not take into account that the WhatsApp Quick Replies are limited to 3, this should split Quick Replies with more than 3 answers into multiple WhatsApp Outputs